tests/e2e/cucumber/steps/ui/spaces.ts

```diff
@@ -82,13 +82,14 @@ When(
 )
 
 When(
-  '{string} adds following users to the project space',
+  '{string} adds following members to the project space',
   async function (this: World, stepUser: string, stepTable: DataTable): Promise<void> {
     const { page } = this.actorsEnvironment.getActor({ key: stepUser })
     const spacesObject = new objects.applicationFiles.Spaces({ page })
-    for (const { user, role } of stepTable.hashes()) {
+    for (const { user, role, kind } of stepTable.hashes()) {
+      const collaborator = kind === 'user' ? this.usersEnvironment.getUser({ key: user }) : this.usersEnvironment.getGroup({ key: user })
```

Comment:
`[this.usersEnvironment.getGroup({](url) key: member })
`or `this.usersEnvironment.getGroup({ key: recipient })`